plugins/content/joomla/joomla.php

```diff
@@ -600,27 +600,6 @@ private function _countItemsInChildren($table, $catid, $data)
 	 */
 	public function onContentChangeState($context, $pks, $value)
 	{
-		$pks = ArrayHelper::toInteger($pks);
-
-		if ($context === 'com_workflow.stage' && $value == -2)
-		{
-			foreach ($pks as $pk)
-			{
-				if (!$this->_canDeleteStage($pk))
-				{
-					return false;
-				}
-			}
-
-			return true;
-		}
```

Comment:
The code should remain, it has nothing to do with the notification, it prevents the deletion of stages when items are assigned.